.env

```diff
@@ -0,0 +1,16 @@
+PORT=3000
+CLIENT_URL=http://127.0.0.1:3000
+
+POSTGRES_HOST=127.0.0.1
+POSTGRES_DB=postgres
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=test123
+
+
+SMPT_HOST=smtp.gmail.com
+SMPT_PORT=587
+SMPT_USER=sendmailer333@gmail.com
+SMPT_PASSWORD=yjkqvugycicqmjdc
+
+JWT_ACCESS_SECRET=efjwhg-ehdsqa-fakd-jajfgfdf3cd
+JWT_REFRESH_SECRET=eaa65921-931b-4e4d-8ce4-1acb545fr
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)